packages/twenty-server/src/engine/core-modules/analytics/analytics.service.ts

```diff
@@ -1,16 +1,20 @@
-import { Injectable, Logger } from '@nestjs/common';
 import { HttpService } from '@nestjs/axios';
+import { Injectable, Logger } from '@nestjs/common';
+
+import { AxiosRequestConfig } from 'axios';
 
 import { EnvironmentService } from 'src/engine/core-modules/environment/environment.service';
 
 type CreateEventInput = {
   type: string;
+  sessionId: string;
   data: object;
 };
 
 @Injectable()
 export class AnalyticsService {
   private readonly logger = new Logger(AnalyticsService.name);
+  private readonly datasource = 'analytics_events';
```

Comment:
Maybe also switch everything to camelCase and singular? That's our current convention in our Postgres database

Reply:
Also I see in Tinybird a table with the singular name has already been created, is this used for testing? you can create branch environments for testing if you need to